### ISSUE TITLE
Conflict with upterm action.

### ISSUE THREAD

**Issue by gtarabat** [webfactory/ssh-agent#124]
In my workflow l use the action [upterm](https://github.com/marketplace/actions/debugging-with-ssh) for debugging purposes.

If I use the `ssh-agent` action before the `upterm` action, the second action throws the error:
```
Creating a new session. Connecting to upterm server ssh://uptermd.upterm.dev:22
no server running on /tmp/tmux-1001/default
Error: 1
```

**Comment by mpdude** [comment]
No, not really. My guess would be that that other action now suddenly tries to authenticate with a key from SSH agent and is not able to deal with a failure, but that’s just a guess.

**Comment by mtamburrano** [comment]
same issue here
